vevestaX/vevesta.py

```diff
@@ -807,17 +807,17 @@ def __EDAForPandas(self, fileName):
         
         #Probability Density Function
         numericDataframe = self.__data.select_dtypes(include='number')  
-        if self.__ycolumn!=None:
+        if self.__Y!=None and (self.__data[self.__Y].dtypes=='int64' or self.__data[self.__Y].dtypes=='object') and self.__data[self.__Y].dtypes!='float64':
             k=1
             fig = plt.figure(figsize=(20,15))
             for i in numericDataframe:
-                if i!=self.__ycolumn:
+                if i!=self.__Y:
                     ax = fig.add_subplot(4,(len(numericDataframe.columns)//4)+1, k)
-                    frequency=self.__data[self.__ycolumn].value_counts()
+                    frequency=self.__data[self.__Y].value_counts()
                     frequency=dict(frequency)
                     frequency=list(frequency.keys())[0:10]
```

Comment:
Have you sorted this to take top 10 frequencies ?

Reply:
By default, value_counts will sort the data by numeric count in descending order.